staging/operator-lifecycle-manager/.github/workflows/e2e-tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.16'
-      - run: make e2e-local E2E_NODES=2 JUNIT_DIRECTORY=./artifacts/
+      - run: make e2e-local E2E_NODES=2 ARTIFACTS_DIR=./artifacts/
```

Comment:
We'll technically need to update o/release as well as we explicitly specify the `JUNIT_DIRECTORY` there. Likely easiest to drop this commit from this PR and tackle at a later date?

Reply:
This was a breaking change then. It's unfortunate we didn't catch it earlier.

Reply:
I called it out when the PR was opened: https://github.com/operator-framework/operator-lifecycle-manager/pull/2432#discussion_r741374284 - imo it's a quick change and not the end of the world as we're between freezes right now and have the flexibility to make these types of changes.